viewing_party/main.py

```diff
@@ -0,0 +1,139 @@
+# wave_01 
+def create_movie(movie_title, genre, rating): 
+    new_movie = {
+                "title": movie_title,
+                "genre": genre,
+                "rating": rating         
+                }
+    for value in new_movie.values(): 
+        if value == None:
+            new_movie = None
+        
+    return new_movie
+
+def add_to_watched(user_data, movie): 
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie): 
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    for movie in user_data["watchlist"]: 
+        if movie["title"] == title: 
+            user_data["watchlist"].remove(movie) 
+            user_data["watched"].append(movie) 
+    
+    return user_data
+
+# wave_02 
+def get_watched_avg_rating(user_data): 
+    total_rating = 0.0
+    average_rating = 0.0
+    watched_movie_amount = 0
+
+    for movie in user_data["watched"]: 
+        total_rating += movie["rating"]
+        watched_movie_amount += 1
```

Comment:
Just a note that you can use `len` to get the number of movies and then you don't need the counter.